packages/kolibri-tools/package.json

```diff
@@ -67,6 +70,7 @@
     "query-ast": "^1.0.5",
     "readline-sync": "^1.4.9",
     "recast": "^0.23.6",
+    "rewire": "^7.0.0",
```

Comment:
rewire before the change had a version of `^6.0.0` Is there any reason why we changed to 7.0.0? This could bring some unintended errors

Reply:
My bad. I'll revert the same.